### DISCUSSION TITLE
Copying files to Nextcloud

### DISCUSSION THREAD

**Discussion by clareclever** [nextcloud/all-in-one#1717] [Questions] [answered]
I have TB’s of data I would like to copy to my local Nextcloud instance. All of the files are currently on hard drives created on Windows. My Nextcloud server is running Ubuntu Server. It should be able to read my external hard drive, my question is however… can I rsync the data from the hard drive directly into a user directory within the Data Directory? For example, instead of copying files over the network (which to copy TB’s of data would take ages) can I use rsync off an external drive to copy them directly?

**Comment by szaimen** [comment, marked as answer]
Hi, as the question looks unrelated to AIO, please ask it at https://help.nextcloud.com. Thanks!
